### ISSUE TITLE
Trees do not render

### ISSUE THREAD

**Issue by wisdom31** [TheGameCreators/GameGuruRepo#339]
Attached are two photos, one from the standalone package without Tree 01 and the test game that had Tree 01. I have tried redoing the standalone package but it still does not work. I am new to the engine and this is the first real map I have made, so forgive any stupid errors.
![snapshot3](https://user-images.githubusercontent.com/42114409/43686022-d2fa6fa4-988c-11e8-888e-69bc0eb49dd7.jpg)
![glitch](https://user-images.githubusercontent.com/42114409/43686023-da80a0c2-988c-11e8-9f95-2ad378e707d8.png)

**Comment by michaeloldman** [comment]
Yeah, been an issue for a while now where some files don't get copied over to the stand-alone.

For now check that all the tree files are there, and if not, copy them over manually. The standalone will be here: Documents\Game Guru Files\My Games.

Reliquia....

**Comment by HierToo** [comment]
#351
